### PR TITLE
Adds upcoming events to the site

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: get_involved.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ permalink: /get_involved/
 toc: false
 ---
 
+## Upcoming Events
+
+**Bug Triage** - every Tuesday and Friday in IRC. Come and participate in real-time. See the
+[triage process](http://docs.pulpproject.org/dev-guide/contributing/bugs.html#triage-process) for
+more details. You can also read through the [triage archives](https://pulpadmin.fedorapeople.org/triage/pulp-dev/?C=N;O=D)
+
+**Community Demo** - Wednesday, April 10, 2019 at [15:00 UTC](https://www.worldtimebuddy.com/?qm=1&lid=100,4487042,3078610&h=100&date=2019-4-10&sln=15-16)
```
Comment (bmbouter) [inline get_involved.md:16]: @CodeHeeler FYI this would be another place to adverise our demos when they schedule/change

File: get_involved.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ permalink: /get_involved/
 toc: false
 ---
 
+## Upcoming Events
+
+**Bug Triage** - every Tuesday and Friday in IRC. Come and participate in real-time. See the
```
Comment (bmbouter) [inline get_involved.md]: This one seems different than the others in that it's a meeting. Can we have a `## Meetings` section added with just this one in it?

Comment (bmbouter) [reply get_involved.md]: Also what about making this the authoritative source on the date/times rather than sending people off of this page?

Comment (dkliban) [reply get_involved.md]: I'll a meetings section above the Upcoming Events. I'll also add the time for triage.

**Comment by bmbouter** [comment]
This looks right to me. I want to check that @CodeHeeler is ok to maintain this as the demo date changes over time before we commit to it.

File: get_involved.md [hunk 3]
```diff
@@ -5,6 +5,22 @@ permalink: /get_involved/
...
+participate in real-time. See the [triage process](http://docs.pulpproject.org/dev-guide/contributing/bugs.html#triage-process) for
+more details. You can also read through the [triage archives](https://pulpadmin.fedorapeople.org/triage/pulp-dev/?C=N;O=D)
+
+## Upcoming Events
+
+**Community Demo** - Wednesday, April 10, 2019 at [15:00 UTC](https://www.worldtimebuddy.com/?qm=1&lid=100,4487042,3078610&h=100&date=2019-4-10&sln=15-16)
+on our [YouTube channel](https://www.youtube.com/channel/UCI43Ffs4VPDv7awXvvBJfRQ). Pulp contributors demonstrate
+the latest changes to Pulp and its plugins. 
+
+**Red Hat Summit** - May 7-9, 2019 in Boston, MA, USA - Pulp is giong to have a booth in Community Central.
```
Comment (daviddavis) [inline get_involved.md]: s/giong/going/

File: get_involved.md [hunk 4]
```diff
@@ -5,6 +5,22 @@ permalink: /get_involved/
...
+
+**Ansible Fest** - September 24-26, 2019 in Atlanta, GA, USA - Pulp is going to have a booth at Ansible Fest.
```
Comment (daviddavis) [inline get_involved.md]: s/Ansible Fest/AnsibleFest/

**Pull request merged**